### PR TITLE
Topic/joseph/v9.2.3

### PR DESCRIPTION
**Pull request by pjosephraj**
Grafana Upgrade from Version 8.5.6 to 9.2.3 with the Insights Branding

### CODE REVIEW

**Review by santosh-at-smarthub — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -148,7 +148,7 @@ shellcheck: $(SH_FILES) ## Run checks for shell scripts.
 
 build-docker-full: ## Build Docker image for development.
 	@echo "build docker container"
-	docker build --tag grafana/grafana:dev .
+	docker build --tag grafana/grafana:dev-f Dockerfile.ubuntu .
```
Comment (santosh-at-smarthub) [inline Makefile:151]:
```suggestion
	docker build --tag grafana/grafana:dev -f Dockerfile.ubuntu .
```

**Review by santosh-at-smarthub — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (pjosephraj) [reply Makefile]: Done, Thank you

**Pull request merged**